app/src/main/java/com/breadwallet/tools/util/PushNotificationService.kt

```diff
@@ -0,0 +1,38 @@
+package com.breadwallet.tools.util
+import android.R
+import android.app.Notification
+import android.app.NotificationManager
+import android.app.PendingIntent
+import android.content.Intent
+import android.media.RingtoneManager
+import android.util.Log
+import androidx.core.app.NotificationCompat
+import com.google.android.gms.tasks.OnCompleteListener
+import com.google.firebase.messaging.FirebaseMessaging
+import com.google.firebase.messaging.FirebaseMessagingService
+import com.google.firebase.messaging.RemoteMessage
+
+
+class PushNotificationService : FirebaseMessagingService() {
+    override fun onNewToken(token: String) {
+        Log.d("push", "Refreshed token: $token")
+
+        // If you want to send messages to this application instance or
```

Comment:
Please take out the commented out 20 - 22

Reply:
Ok, noted!

Reply:
done